src/main/java/com/google/api/codegen/discovery/transformer/go/GoSampleMethodToViewTransformer.java

```diff
@@ -96,7 +96,10 @@ private SampleView createSampleView(SampleTransformerContext context) {
       fieldVarNames.add(requestBodyVarName);
     }
 
-    if (methodInfo.isPageStreaming()) {
+    // The Go client only considers methods whose verb is "GET" to be page
```

Comment:
Unless you're asking like metaphorically, in which case I don't think so, methods whose verb is "POST" can also be paging